tests/test_experiment.py

```diff
@@ -466,13 +467,13 @@ def test_validate_args(builder: str, kwargs, fails, err_msg):
         (
             "esnmebel=True",
             False,
-            {},
+            ("esnmebel", "True"),
         ),
     ],
 )
 def test_parse_kwargs(kwargs, fails, expected):
```

Comment:
```suggestion
def test_parse_kwarg(kwargs, fails, expected):
```

Just to be ridiculously picky.